.github/workflows/action.yml

```diff
@@ -10,19 +10,33 @@ jobs:
     - name: Run Shellcheck
       uses: bewuethr/shellcheck-action@v2
 
-    - name: Install eksctl and create EKS cluster
+    - name: Install eksctl
       run: |
-        chmod +x ./eks/install_eks.sh
-        ./eks/install_eks.sh
+        chmod +x ./eksctl/install.sh
+        ./eksctl/install.sh
+  
+    - name: Configure AWS credentials
+      run: |
+        chmod +x ./eks/aws-credentials.sh
+        ./eks/aws-credentials.sh
       env:
         ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
         SECRET_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         REGION: ${{ secrets.AWS_DEFAULT_REGION }}
 
+    - name: Create EKS cluster
+      run: |
+        sed 's|region: us-east-1|region: '$REGION'|' -i ./eks/cluster.yml
+        sed 's|version: 1.16|version: '$VERSION'|' -i ./eks/cluster.yml
+        eksctl create cluster -f ./eks/cluster.yml
+      env:
+        VERSION: ${{ secrets.VERSION }}
```

Comment:
Can we hardcode the version & region instead?

Reply:
Or can we have a script that exports these as env variables
This script can be present in ./eks/env.sh